content/js/record.md

```diff
@@ -1,117 +1,99 @@
-# Saving a record
-
-You can save a public note to server as follow.
-
-``` javascript
-skygear.publicDB.save(new Note({
-    'content': 'Hello World!'
-})).then((record) => {
-    console.log(record);
-}, (error) => {
-    console.log(error);
-});
-```
-
-To understand the code above, you need you know what is _container_, _database_
-and _record_.
-
+<a name="basic-crud"></a>
 # Container
 
 Container is the uppermost layer of skygear. In practice,
 `import skygear from 'skygear'` will give you a container instance at variable
-skygear. In most case you will only need one instance of containter.
+skygear. In most case you will only need one instance of container.
 
 The first things you need to interact with container is setting `endPoint` and
-`accessToken`.
+`apiKey`.
 
 ``` javascript
-JSSkygear.endPoint = 'http://mydeployment.dev/api/';
-JSSkygear.configApiKey('mysecrect');
+import skygear from 'skygear';
+// or in the browser with ECMAScript 5 just use window.skygear
+
+skygear.config({
+    'endPoint': 'https://<your-app-name>.staging.skygeario.com',
+    'apiKey': '<your-api-key>'
+}).then((container) => {
+    console.log(container);
+}, (error) => {
+    console.error(error);
+});
 ```
 
 # Database
 
-You will provide with private and public database.
+You will be provided with a private and a public database.
 
-- Everything in private database is truely private, regardless of what access
+- Everything in the private database is truly private, regardless of what access
 control entity you set to the record.
-- Record saved at public database is defualt public. To control the access, you
-may set difference access control entity to the record.
+- Record saved at public database is by default public. To control the access, you
+may set different access control entity to the record.
+- The database objects can be accessed with `skygear.publicDB` and
+`skygear.privateDB`.
 
+<a name="record-record"></a>
 # Record
 
-- `Record` must have type.
-- `Record` is a key-value data object that store at _database_.
+- `Record` must have a type.
+- `Record` is a key-value-pair data object that is stored at _database_.
 - `Record` will belong to the currently logged in user.
+- `Record` object has unique `id` (a string combination of record type and uuid).
 
-
-# Defining a record type
-
-You will design different record type to model your app. Just like define table
-in SQL.
+You can design different `Record` type to model your app. Just like defining
+tables in SQL.
 
 ``` javascript
 const Note = skygear.Record.extend('note');
 const Blog = skygear.Record.extend('blog');
+
+let note = new Note({ "content": "Hello World" });
+// { id: "...", recordType: "note", access: [Object object] }
 ```
 
-# Modify a record
+# (Create) Save a record
```

Comment:
I don't know if we want to have the brace `(Create)`. It look duplicate to me and title should be as short as possible?